src/mb_multi_external_links/index.ts

```diff
@@ -0,0 +1,197 @@
+            editor.handleUrlBlur = patchedOnBlur;
+            editor.handleUrlChange = patchedOnChange;
+        },
+        disable(): void {
+            LOGGER.debug('Disabling link splitter');
+            editor.handleUrlBlur = originalOnBlur;
+            editor.handleUrlChange = originalOnChange;
+        },
+        setEnabled(enabled: boolean): void {
+            if (enabled) {
+                this.enable();
+            } else {
+                this.disable();
+            }
+        },
+        toggle(): void {
+            this.setEnabled(editor.handleUrlBlur === originalOnBlur);
+        },
+    };
+}
+
+function insertCheckboxElements(editor: ExternalLinks, checkboxElmt: HTMLInputElement, labelElmt: HTMLLabelElement): void {
+    // Adding the checkbox beneath the last input element would require constantly
+    // removing and reinserting while react re-renders the link editor. Instead,
+    // let's just add it outside of the table and align it with JS.
+    editor.tableRef.current.after(checkboxElmt, labelElmt);
+    const lastInput = getLastInput(editor);
+    const marginLeft = lastInput.offsetLeft + (lastInput.parentElement?.offsetLeft ?? 0);
+    checkboxElmt.style.marginLeft = `${marginLeft}px`;
+}
+
+async function run(windowInstance: Window): Promise<void> {
+    // This element should be present even before React is initialized. Checking
+    // for its existence enables us to skip attempting to find the link input on
+    // edit pages that don't have external links, without having to exclude
+    // specific pages.
+    const editorContainer = qsMaybe<HTMLElement>('#external-links-editor-container', windowInstance.document);
+    if (!editorContainer) return;
+
+    // We might be running before the release editor is loaded, so retry a couple of times
+    // until it is. We could also just have a different @run-at or listen for the window
+    // load event, but if the page takes an extraordinary amount to load e.g. an image,
+    // the external links editor may be ready long before we run. We want to add our
+    // functionality as soon as possible without waiting for the whole page to load.
+    const editor = await retryTimes(() => getExternalLinksEditor(windowInstance.MB), 100, 50);
+    const splitter = createLinkSplitter(editor);
+    const [checkboxElmt, labelElmt] = createPersistentCheckbox('ROpdebee_multi_links_no_split', "Don't split links", () => {
+        splitter.toggle();
+    });
+    splitter.setEnabled(!checkboxElmt.checked);
+    insertCheckboxElements(editor, checkboxElmt, labelElmt);
+}
+
+function onIframeAdded(iframe: HTMLIFrameElement): void {
+    LOGGER.debug(`Initialising on iframe ${iframe.src}`);
+    const iframeWindow = iframe.contentWindow;
+    if (!iframeWindow) return;
+
+    function runInIframe(): void {
+        run(iframeWindow!)
+            .catch((err) => {
+                LOGGER.error('Something went wrong', err);
+            });
```

Comment:
```suggestion
        run(iframeWindow!)
            // TODO: Replace this by `logFailure`
            .catch((err) => {
                LOGGER.error('Something went wrong', err);
            });
```
Also depending on #451